src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/WithdrawalTest.kt

```diff
@@ -279,18 +279,6 @@ class WithdrawalTest : IntegrationTestBase() {
               decision = PlacementApplicationDecision.ACCEPTED,
             )
 
-            createPlacementApplication(
-              application,
-              DateSpan(now(), duration = 2),
-              decision = PlacementApplicationDecision.WITHDRAW,
-            )
-
-            createPlacementApplication(
-              application,
-              DateSpan(now(), duration = 2),
-              decision = PlacementApplicationDecision.WITHDRAWN_BY_PP,
-            )
```

Comment:
I think for this test we need at least one placement application that is legitimately withdrawn (i.e. where the new isWithdrawn boolean is set to true). This shouldn't exist in the response from the API, so it's just a matter of creating a placement request here (essentially folding down the original two withdrawn placements applications with different decisions types into a single placement application with the new boolean set to true).

Also, could you please review the javadoc on the test function and update it to reflect this